refactor(home): update home layout

mattnolting opened the pull request.

eriknelson commented:
Merge conflict on this one

ibolton336 commented:
![image](https://user-images.githubusercontent.com/11218376/64567649-b5926900-d326-11e9-8b3d-5533bf233c2e.png)

We received some initial help on the card component format from UXD. @mcoker and @kyle gave us mockups which we implemented for these cards. 

I know the most recent feedback requested a change of the hierarchy  ( https://marvelapp.com/66cj5ga/screen/60982350 ) , but I think we might have to give this more thought. Feels like a step back IMO

mattnolting commented:
@ibolton336 Looks like the className changed and the cards lost styling. Updated that and added support for `.pf-c-page__header` background-color. 

@eriknelson conflicts resolved.
